config/default.cjs

```diff
@@ -1,20 +1,20 @@
 module.exports = {
```

Comment:
I think that config as one of data sources should be on TS. Looks like it was solver for mailing project in [this PR](https://github.com/plyo/mailing/pull/206/files#diff-3f04c0bcd9b075fa014d3700396c2b940c8fcfcd5350a79e43082fbaa55a85ebR10). Please talk to @Minoga, I remember there were some problems with it

Reply:
Ready and converted to TS